gnuplot/src/color.rs

```diff
@@ -0,0 +1,413 @@
+pub use self::ColorType::*;
+use crate::util::OneWayOwned;
+use std::fmt::Display;
+
+pub trait IntoColor<T>: Into<ColorType<T>> + Clone {}
```

Comment:
Is this used for anything, I can't seem to find uses of it.

Reply:
Good spot: this was used previously, but I found an implementation that doesn't need it. Removed.